experimental/Schemes/elliptic_surface.jl

```diff
@@ -1021,21 +1021,25 @@ function section(X::EllipticSurface, P::EllipticCurvePoint)
   return _section(X, P)
 end
 
-function _section(X::EllipticSurface, P::EllipticCurvePoint)
-  @vprint :EllipticSurface 3 "Computing a section from a point on the generic fiber\n"
+function _section_on_weierstrass_ambient_space(X::EllipticSurface, P::EllipticCurvePoint)
   S0,incS0 = weierstrass_model(X)
   X0 = codomain(incS0)
   if P[3] == 0
     # zero section
     V = X0[1][3]
     (z,x,t) = coordinates(V)
-    PX = IdealSheaf(X0, V, [x,z])
-  else
-    U = X0[1][1]
-    (x,y,t) = coordinates(U)
-    b = P
-    PX = ideal_sheaf(X0,U,[OO(U)(i) for i in [x*denominator(b[1])(t)-numerator(b[1])(t),y*denominator(b[2])(t)-numerator(b[2])(t)]])
+    return IdealSheaf(X0, V, [x,z])
   end
+  U = X0[1][1]
+  (x,y,t) = coordinates(U)
+  b = P
+  return ideal_sheaf(X0,U,[OO(U)(i) for i in [x*denominator(b[1])(t)-numerator(b[1])(t),y*denominator(b[2])(t)-numerator(b[2])(t)]])
+end
+
+function _section(X::EllipticSurface, P::EllipticCurvePoint)
```

Comment:
We could use the strict transform ideal sheaf or the prime ideal sheaf here instead. 
... in the past this somehow killed performance but with the current changes it is maybe okay again?

Reply:
At least the creation of the object should run more or less instantly now, yes.

Reply:
Because `strict_transform` returns a `StrictTransformIdealSheaf`?